### PR TITLE
Geneva, docs, and fixes

### PR DESCRIPTION
**Pull request by rafsalas19**
- Geneva can now be used w/ linux services
- Arm Template fix
- Documentation update

### CODE REVIEW

File: deploy_managed_infra/managed_infra_template.json [hunk 1]
```diff
@@ -99,92 +99,92 @@
...
                         "record": "average_dcgm_gpu_temp",
                         "enabled": true,
-                        "expression": "avg(dcgm_power_usage) by (instance, subscription, cluster, job_id)"
+                        "expression": "avg(dcgm_gpu_temp) by (instance, subscription, cluster, job_id)"
                     },
                     {
                         "record": "max_dcgm_gpu_temp",
                         "enabled": true,
-                        "expression": "avg(dcgm_power_usage) by (instance, subscription, cluster, job_id)"
+                        "expression": "avg(dcgm_gpu_temp) by (instance, subscription, cluster, job_id)"
```
Comment (moes1) [inline deploy_managed_infra/managed_infra_template.json:107]: should this expression be max(dcgm_gpu_temp) ?

File: deploy_managed_infra/managed_infra_template.json [hunk 2]
```diff
@@ -99,92 +99,92 @@
...
                         "record": "average_dcgm_memory_temp",
                         "enabled": true,
-                        "expression": "avg(dcgm_power_usage) by (instance, subscription, cluster, job_id)"
+                        "expression": "avg(dcgm_memory_temp) by (instance, subscription, cluster, job_id)"
                     },
                     {
                         "record": "max_dcgm_memory_temp",
                         "enabled": true,
-                        "expression": "avg(dcgm_power_usage) by (instance, subscription, cluster, job_id)"
+                        "expression": "avg(dcgm_memory_temp) by (instance, subscription, cluster, job_id)"
```
Comment (moes1) [inline deploy_managed_infra/managed_infra_template.json:122]: I think avg should be max here?

File: deploy_managed_infra/managed_infra_template.json [hunk 3]
```diff
@@ -99,92 +99,92 @@
...
                         "record": "average_dcgm_sm_clock",
                         "enabled": true,
-                        "expression": "avg(dcgm_power_usage) by (instance, subscription, cluster, job_id)"
+                        "expression": "avg(dcgm_sm_clock) by (instance, subscription, cluster, job_id)"
                     },
                     {
                         "record": "max_dcgm_sm_clock",
                         "enabled": true,
-                        "expression": "avg(dcgm_power_usage) by (instance, subscription, cluster, job_id)"
+                        "expression": "avg(dcgm_sm_clock) by (instance, subscription, cluster, job_id)"
```
Comment (moes1) [inline deploy_managed_infra/managed_infra_template.json:137]: same as above

File: linux_service/README.md [hunk 4]
```diff
@@ -18,6 +18,8 @@ Three launch methods provided:
 3. Launch exporters and an [Azure Monitor](../docs/AzureMonitorAgent.md) publisher.
    - Before launch you must modify the "azure_monitor_agent_config" section of [publisher_config](../src/worker/publisher/config/publisher_config.json) file with the Azure Monitor workspace connection string.
 
+There is one additional method for internal Msft use that exports to Geneva. This method is similar to Azure Monitor method but uses a Geneva agent container to export. Reference the [Moneo Geneva Docs](../docs/GenevaAgent.MD). Ensure all prequisites are met.
```
Comment (moes1) [inline linux_service/README.md:21]: Is this needed in the public documentation?

File: linux_service/configure_service.sh [hunk 5]
```diff
@@ -1,6 +1,21 @@
...
+if [[ -n $PublisherMethod ]]; then
+    if [ "$PublisherMethod" == "geneva" ] || [ "$PublisherMethod" == "azure_monitor" ]; then
+        echo "PublisherMethod is valid: $PublisherMethod"
+    else
+        echo "PublisherMethod $PublisherMethod is not one of the valid choices {azure_monitor, geneva}."
+        exit 1
+    fi
+fi
+
+MONEO_PATH=/home/rafsalas/Moneo
```
Comment (moes1) [inline linux_service/configure_service.sh:18]: Do we want this path hardcoded?

File: linux_service/README.md [hunk 6]
```diff
@@ -39,11 +41,12 @@ Below are the prereqs needed:
...
 ### Configuration and Installation ###
 
-Configuration/Installation is only required once. Afte that is complete the Linux services can be started and stopped as desired.
+Configuration/Installation is only required once. After that is complete the Linux services can be started and stopped as desired.
 
 1. Configuration and installation of the Linux service is done with the following command:
    ```parallel-ssh -i -t 0 -h hostfile "sudo /opt/azurehpc/tools/Moneo/linux_service/configure_service.sh"```
-   - If You will only be launching the exporters without AZ monitor or Managed Prometheus Continue to the Launch Services section else continue.
+     - Note: If using Azure monitor or Geneva add an extra argument "./start_moneo_services.sh azure_monitor"  or "./configure_service.sh geneva" respectively.
+     - Note: Geneva authentication is user managed identity "umi"by default, you can choose to change to "cert" method by modifiying [the start script](./configure_service.sh) "PUBLISHER_AUTH" variable.
```
Comment (jeseszhang1010) [inline linux_service/README.md:49]: NIT: space btw "umi"by

File: linux_service/configure_service.sh (hunk 5 above)
Comment (rafsalas19) [reply linux_service/configure_service.sh:18]: ah good catch let me address

File: deploy_managed_infra/managed_infra_template.json (hunk 2 above)
Comment (rafsalas19) [reply deploy_managed_infra/managed_infra_template.json:122]: let me address these

File: linux_service/README.md (hunk 4 above)
Comment (rafsalas19) [reply linux_service/README.md:21]: Its easier to maintain the instructions here. All that is being shared is how to configure Moneo/VMSS.

File: deploy_managed_infra/managed_infra_template.json (hunk 3 above)
Comment (rafsalas19) [reply deploy_managed_infra/managed_infra_template.json:137]: addressed

File: deploy_managed_infra/managed_infra_template.json (hunk 2 above)
Comment (rafsalas19) [reply deploy_managed_infra/managed_infra_template.json:122]: addressed all instances

File: linux_service/configure_service.sh (hunk 5 above)
Comment (rafsalas19) [reply linux_service/configure_service.sh]: addressed

File: linux_service/README.md (hunk 6 above)
Comment (rafsalas19) [reply linux_service/README.md]: addressed

File: deploy_managed_infra/managed_infra_template.json (hunk 1 above)
Comment (rafsalas19) [reply deploy_managed_infra/managed_infra_template.json:107]: addressed

**Review by RyoYang — APPROVED** [review]
LGTM!

**Pull request merged**